### ISSUE TITLE
Jpeg XL colors issue and possible crash issue

### ISSUE THREAD

**Issue by Funkmiser** [dlemstra/Magick.NET#1212]
### Magick.NET version

v11.2.0

### Environment (Operating system, version and so on)

Windows 11 x64 Version 21H2 (Build 22000.739)

### Description

[Test crash image and conversion.zip](https://github.com/dlemstra/Magick.NET/files/9048472/Test.crash.image.and.conversion.zip)
[Wrong colors images.zip](https://drive.google.com/file/d/1-vQf05lzci1N9R4g4hYgoBoiTR6S8AC7/view?usp=sharing)
The wrong colors zip is around 290MB which is why I had to use Google Drive for the upload.

I am using ImageGlass Kobe v8.6.7.13 x64 to view the jpeg xl files, and it uses the Magick.Net v11.2.0 dlls. I can't tell whether the crashing issue is from ImageGlass or Magick.Net but I believe the colors issue is a Magick.Net thing. Both of the png files from the zips above were converted from TIFFs using ImageMagick 7.1.0-40 mogrify default parameters. They were then converted to jxl using default parameters with the cjxl.exe v0.6.1 file. 
The crash jxl image from the Test crash image zip seems to crash ImageGlass a second or two after attempting to open it, which is why I can't tell whether it's an ImageGlass issue or a Magick.Net issue.
For the colors issue, opening the jxl file in the Wrong colors zip shows an image that has the wrong colors compared to the png inside the same zip file. Like mentioned above these pngs were converted from TIFFs so is it an EXIF issue? I will mention that both the TIFF image and the png image didn't shows colors like the jxl image.

![Screenshot 2022-07-05 132109](https://user-images.githubusercontent.com/53496265/177391199-90951420-4f7e-4ef3-a50a-a871c9f01eeb.png)
This is a screenshot of the colors I'm seeing.

For the crashing I have opened an ImageGlass issue [https://github.com/d2phap/ImageGlass/issues/1381](https://github.com/d2phap/ImageGlass/issues/1381)

### Steps to Reproduce

1. Take the pngs from the two zip files above and convert them to jxls using cjxl.exe [image].png [image].jxl
2. Open them and notice the issue(s)

**Comment by Funkmiser** [comment]
For some further information, I was messing around with the images and using djxl.exe to check the reference decoding. The crash image decodes fine, but the colors of the decoded Software Poster 2 image that shows wrong colors is slightly off, but not as off as when viewed using ImageGlass. Seems to be slightly more blue/dark than the regular but not super dark like ImageGlass shows.

![Screenshot 2022-07-06 021123 blue](https://user-images.githubusercontent.com/53496265/177491340-f8759492-69d6-4e54-9038-cd84b2db7fef.png)
Here is the decoded image using djxl.exe to jpg, slightly more blue/dark. It's the same when using djxl.exe to convert to png.

![Screenshot 2022-07-06 021237 regular](https://user-images.githubusercontent.com/53496265/177491466-eb040c60-9fb9-46ee-9a5f-cab606bc5452.png)
For visual reference, here's what ImageGlass shows for the original png.

**Comment by Funkmiser** [comment]
This gets more and more curious, I added more info in the libjxl issue I created but when using the older v0.6.1 djxl.exe to get a jpg of the incorrect -d 1 version, the result is more blue. But if I get a libjxl artifact build of a newer/latest version at the time and do the same thing with the newer djxl.exe the resulting jpg is still incorrect but the colors are more grey/dark than blue. I even did a git clone and built the programs using msys2 and djxl was still wrong.

Yet it's still the same with -d 0 where the jxl and djxl jpg are both correct compared to the png.

Oh, as a side note someone mentioned Irfanview showing the jxl correctly and it does in a sense. When viewing the png and jxl using Irfanview the views match each other, but compared to ImageGlass the images are way darker.

I originally thought this might be an EXIF data issue since I was converting from a TIFF to a png before the jxl conversion but then dismissed it because -d 0 was showing up correctly, but who knows at this point. 🙃
